community/SIG-GOVERNANCE.md

```diff
@@ -19,3 +19,19 @@ SIGs are held accountable to the Maintainers.
 * Meetings should be open to the community unless there is a specific reason for keeping it private.
 * Meeting recordings and/or minutes should be published within 24 hours of the meeting.
 For embargoed decisions, the decisions must be made public as soon as the embargo is lifted.
+
+### Creation
+
+* A new SIG could be in the formation phase by sharing the proposal with the maintainer meeting and receiving unanimous agreement in the maintainer meeting.
+* A new SIG formation phrase should last for more than 3 months and also meet the following milestones:
+
+  * The formation SIG has a clear and actionable goal defined
+  * The formation SIG has interested members from more than 3 organizations
+  * The formation SIG has conducted several (3+) routine meetings to make substantial progress in its agenda
+* After the formation phase, the formation SIG team should present to the maintainer meeting about the current progress. 
+* The unanimous agreement among maintainers could turn the formation SIG into the official SIG. With any disagreement, the proposed SIG will remain in the formation phase.
+
+### Deprecation
+
+* The maintainer meeting should review SIGs each quarter for their activities in the maintainer meeting.
```

Comment:
I think this is too frequent for maintainer to review the SIG, with the low number of maintainers will be hard to find a maintainer 4 times in a year to help review

Reply:
This doesn't add extra burden to the existing approved requirement documented here - https://github.com/argoproj/argoproj/blob/master/community/SIG-GOVERNANCE.md?plain=1#L18

> Each SIG must conduct a quarterly review of their past and planned activities with the Maintainers. 

The review could be async. It is more focusing the progress.